src/elements/content-sidebar/activity-feed/common/delete-confirmation/DeleteConfirmation.js

```diff
@@ -42,7 +42,7 @@ class DeleteConfirmation extends React.Component<Props> {
     };
 
     render() {
-        const { message, onDeleteCancel, onDeleteConfirm, ...rest } = this.props;
+        const { isOpen, message, onDeleteCancel, onDeleteConfirm, ...rest } = this.props;
```

Comment:
This is to avoid passing `isOpen` down as a prop to the underlying `div` on the `FocusTrap` component of `Overlay`

Reply:
Nice, this solves one of the console errors we've been seeing related to unsupported props in the DOM.